src/pages/feed/RecentsFeed/RecentsFeed.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { push } from 'react-router-redux'
+import { connect } from 'react-redux'
+import Icon from '@santiment-network/ui/Icon'
+import { ProjectSelector } from '../../../ducks/SANCharts/Header'
+import { SidecarItems } from '../../../ducks/SANCharts/ChartSidecar'
+import styles from './RecentsFeed.module.scss'
+
+const RecentsFeed = ({ onSlugSelect }) => {
+  return (
+    <div className={styles.container}>
+      <div>
+        <ProjectSelector
+          trigger={() => (
+            <div className={styles.search}>
+              Explore assets
+              <Icon type='search' className={styles.searchIcon} />
+            </div>
+          )}
+        />
+      </div>
+      <div className={styles.scrollable}>
+        <SidecarItems
+          onSlugSelect={onSlugSelect}
+          onProjectClick={onSlugSelect}
+          classes={styles}
+          params={{
+            gainersTitle: 'Gainers and loosers'
+          }}
+        />
+      </div>
+    </div>
+  )
+}
+
+const mapDispatchToProps = dispatch => ({
+  onSlugSelect: ({ slug }) => {
+    dispatch(push('/projects/' + slug))
```

Comment:
Could you check this again, pls. Doesn't work for me, when I choose asset from popup